src/ui/hooks/useReplayClient.ts

```diff
@@ -63,4 +63,6 @@ export default function useReplayClient() {
       }
     };
   }, []);
+
+  return replayClient;
```

Comment:
Eh, it's still only being called in one place - I just moved it up to the actual Next page level so it can be passed into the setup logic.  I don't think we have a reason to call it anywhere else.

Reply:
I know. I agree that we don't. My concern here is that adding a return value _implies_ that it's meant to be the way (or at least _a valid way_) to get the client– and it's not. It's specifically only meant to _configure_ the client. Does that clarify my concern..?